base_layer/wallet/src/output_manager_service/tasks/txo_validation_task.rs

```diff
@@ -318,6 +319,13 @@ where
                 )
                 .await?;
             }
+            for unmined_output in unmined {
+                if unmined_output.status == OutputStatus::UnspentMinedUnconfirmed {
```

Comment:
Maybe add an info log message here?

Reply:
If we add anything info is wrong here, perhaps trace
Its fixing the state of the db here